topics/imaging/tutorials/process-image-bioimageio/tutorial.md

```diff
@@ -0,0 +1,182 @@
+- HPACellSegmentationModel.pt
+- MitochondriaEMSegmentationBoundaryModel.pt
+
+## Available BioImage.IO models in Galaxy
+
+| Model name | Task | Imaging modality | Sample / species | Link |
+|------------|------|------------------|------------------|------|
+| PlatynereisEMnucleiSegmentationBoundaryModel | Nuclei segmentation | Electron microscopy | Platynereis | [View model](https://bioimage.io/#/?id=platynereisemnucleisegmentationboundarymodel) |
+| PlatynereisEMcellsSegmentationBoundaryModel | Cell segmentation | Electron microscopy | Platynereis | [View model](https://bioimage.io/#/?id=platynereisemcellssegmentationboundarymodel) |
+| livecellsegmentationboundarymodel | Live cell segmentation | Phase-contrast Microscopy | Various cell types | [View model](https://bioimage.io/#/?id=livecellsegmentationboundarymodel) |
+| HyLFM-Net-stat | Light field reconstruction | Light field and Fluorescence light microscopy | Zebrafish | [View model](https://bioimage.io/#/?id=hylfm-net-stat) |
+| 3DUNetArabidopsisApicalStemCells | Stem cell segmentation | Confocal / light sheet | Arabidopsis root | [View model](https://bioimage.io/#/?id=3dunetarabidopsisapicalstemcells) |
+| CovidIFCellSegmentationBoundaryModel | Cell segmentation | Fluorescence light microscopy | Infected human cells | [View model](https://bioimage.io/#/?id=covidifcellsegmentationboundarymodel) |
+| nucleisegmentationboundarymodel | Nucleus segmentation | Fluorescence light microscopy| Generic / various | [View model](https://bioimage.io/#/?id=nucleisegmentationboundarymodel) |
+| HPANucleusSegmentation | Nucleus segmentation | Immunofluorescence | Human Protein Atlas | [View model](https://bioimage.io/#/?id=hpanucleussegmentation) |
+| NeuronSegmentationInEM | Neuron segmentation | Electron microscopy | Brain tissue | [View model](https://bioimage.io/#/?id=neuronsegmentationinem) |
+| HPACellSegmentationModel | Cell segmentation | Immunofluorescence | Human Protein Atlas | [View model](https://bioimage.io/#/?id=hpacellsegmentationmodel) |
+| MitochondriaEMSegmentationBoundaryModel | Mitochondria segmentation | Electron microscopy | Human | [View model](https://bioimage.io/#/?id=mitochondriaemsegmentationboundarymodel) |
+
+## Model-specific example
+
+This example illustrates the type of information that is both useful for understanding the model’s biological context and necessary for using the Galaxy tool — specifically, the input axes and input size parameters.
+
+<details>
+<summary><strong>🧬 NucleiSegmentationBoundaryModel</strong></summary>
+
+This model segments nuclei in fluorescence microscopy images. It predicts boundary maps and foreground probabilities for nucleus segmentation, primarily in images stained with DAPI. The outputs are designed to be post-processed with methods such as Multicut or Watershed to achieve instance-level segmentation.
+
+- **Imaging modality**: Fluorescence microscopy  
+- **Task**: Nucleus segmentation (boundary-aware)  
+- **Input axes**: `bcyx`  
```

Comment:
What does the `b` axis stand for? Never saw that before.

Might also be worth adding a small explanation of `bcyx` to the text.